src/test/java/com/aws/greengrass/clientdevices/auth/iot/registry/CertificateRegistryTest.java

```diff
@@ -115,4 +120,29 @@ void GIVEN_offline_initialization_WHEN_isCertificateValid_THEN_throw_exception()
         assertThrows(CloudServiceInteractionException.class, () -> registry.isCertificateValid(mockCertPem));
         assertThrows(CloudServiceInteractionException.class, () -> registry.getIotCertificateIdForPem(mockCertPem));
     }
+
+    @Test
+    void GIVEN_certRegistry_WHEN_refreshRegistry_THEN_stale_entries_removed() {
+        String certHash1 = "certHash1";
+        String certHash2 = "certHash2";
+        String certHash3 = "certHash3";
+
+        CertificateEntry validEntry = new CertificateEntry(Instant.now().plusSeconds(10L),
```

Comment:
should use same base instant for all entries---the `Instant.now()` two lines below could be different than the one here.  same goes for other test class